domain/src/test/java/com/distillery/android/domain/FakeToDoRepositoryTest.kt

```diff
@@ -73,6 +75,34 @@ class FakeToDoRepositoryTest {
         subscription.cancel()
     }
 
+    @Test
+    fun `repository delete values after 1_5 seconds`() = runBlocking {
+        // given we subscribed to repository changes
```

Comment:
Minor: I think such verbose comments aren't necessary
```suggestion
        // GIVEN
```